src/components/banner/Banner.tsx

```diff
@@ -0,0 +1,57 @@
+import { Grid, Link, Paper } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import * as React from 'react';
+import '../../styles.css';
+
+interface State {
+  bannerIsHidden: boolean
+}
+
+class Props {
+  public message: any
```

Comment:
This isn't used at all

Reply:
I know, it's a placeholder while I figure out the message passing thing I left a comment about above.